### DISCUSSION TITLE
selectedRowsQuery get row items NOT WORKING

### DISCUSSION THREAD

**Discussion by rogerad** [rappasoft/laravel-livewire-tables#336] [General]
Hi 
i am trying to access the selected rowsquery to export to excel   -  selectedRowsQuery 

the docs says
access to $this->selectedRowsQuery which is a Builder instance of the selected rows.

The rows are selected but i **cannot get the data** from each row
how do i do this >

**Comment by rappasoft** [comment]
`$this->selectedRowsQuery()->get()`

<img width="553" alt="Screen Shot 2021-06-10 at 19 02 22" src="https://user-images.githubusercontent.com/6026504/121607681-6fe46200-ca1e-11eb-8ceb-75153ec2fe19.png">

**Comment by rogerad** [comment]
Thank you 

 It works

 

Roger

**Reply by rappasoft** [reply]
Please mark the accepted answer for everyone else. Thanks.

**Reply by rogerad** [reply]
I cant find a spot when to make accepted so i am closing thank you
You can't mark a threaded comment in response to a comment as the answer to a discussion. You also can't mark a minimized comment as the answer to a discussion.

    On GitHub, navigate to the main page of the repository.
